components/views/auth/ResetPassword.tsx

```diff
@@ -0,0 +1,109 @@
+import { useState, FormEvent, useEffect } from 'react'
+import useForm from '../../../utilities/hooks/useForm'
+import { FormProps } from '../../../utilities/types/formTypes'
+import { VALID_PASSWORD } from '../../../utilities/validations/regexValidators'
+import { FormRow, PasswordField } from '../../form'
+import { BiErrorAlt } from 'react-icons/bi'
+
+import styles from './AuthPages.module.scss'
+
+const INITIAL_STATE = {
+    newPassword: { value: '', error: '' },
+    confirmPassword: { value: '', error: '' }
+} as FormProps
+
+const PASSWORD_ERROR_MESSAGE = 'Password does not meet requirements'
```

Comment:
This needs to be specific to the requirements of the password instead

Reply:
done